bugsnag-android-core/src/main/java/com/bugsnag/android/Configuration.java

```diff
@@ -64,7 +59,7 @@
     private final Collection<BeforeSendSession> sessionCallbacks = new ConcurrentLinkedQueue<>();
 
     private String codeBundleId;
-    private String notifierType;
+    private String appType = "android";
```

Comment:
This is pre-existing behaviour that has been refactored as part of this PR - my understanding was that this could also affect how the pipeline processes events, although I'm not sure exactly how.

Reply:
Oh ok, I thought the previous behaviour was that it was empty. I'm not sure the pipeline does anything with this field but since this PR is just about renaming options I'm happy for the behaviour to be figured out another time.